client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,986 @@
+					copy: translate( 'Free for the first year' ),
+					mobileCopy: translate( 'Free domain for the first year' ),
+				},
+				planTwo: {
+					included: true,
+					copy: translate( 'Free for the first year' ),
+					mobileCopy: translate( 'Free domain for the first year' ),
+				},
+				planThree: {
+					included: true,
+					copy: translate( 'Free for the first year' ),
+					mobileCopy: translate( 'Free domain for the first year' ),
+				},
+			},
+			{
+				featureName: translate( 'SEO' ),
+				tooltip: translate(
+					'SEO (Search Engine Optimization) helps your site rank higher, and more accurately in search engines, so more people can find your site.'
+				),
+				planOne: {
+					included: true,
+					copy: translate( 'Simple' ),
+					mobileCopy: translate( 'Simple SEO' ),
+				},
+				planTwo: {
+					included: true,
+					copy: translate( 'Professional for better ranking' ),
+					mobileCopy: translate( 'Professional SEO' ),
+				},
+				planThree: {
+					included: true,
+					copy: translate( 'Professional for better ranking' ),
+					mobileCopy: translate( 'Professional SEO' ),
+				},
+			},
+			{
+				featureName: translate( 'Customer support' ),
+				tooltip: translate(
+					'Keep moving forward with your site at any time. Our Happiness Engineers will help you via email, live chat, or 24/7 priority live chat, depending on the plan you choose.'
+				),
+				planOne: {
+					included: true,
+					copy: translate( 'Live chat and email' ),
+					mobileCopy: translate( '100GB Storage' ),
+				},
+				planTwo: {
+					included: true,
+					copy: translate( '24/7 Priority live chat' ),
+					mobileCopy: translate( '24/7 Priority live chat' ),
+				},
+				planThree: {
+					included: true,
+					copy: translate( '24/7 Priority live chat' ),
+					mobileCopy: translate( '24/7 Priority live chat' ),
+				},
+			},
+			{
+				featureName: translate( 'Storage' ),
+				tooltip: translate(
+					'With increased storage space, you’ll be able to upload more images, audio, and documents to your website. On the Premium, Business, and eCommerce plans, you can upload videos, too.'
+				),
+				planOne: { included: true, copy: translate( '10GB storage' ), mobileCopy: null },
+				planTwo: { included: true, copy: translate( '200GB storage' ), mobileCopy: null },
+				planThree: {
+					included: true,
+					copy: translate( '200GB storage' ),
+					mobileCopy: translate( '200GB storage' ),
+				},
+			},
+		],
+		Limited: [
+			{
+				featureName: translate( 'Install plugins' ),
+				tooltip: translate(
+					'Plugins are like apps for your site. They add new functionality and features to expand your site. There are over 58,000+ WordPress plugins available for anything you need.'
+				),
+				planOne: { included: false, copy: translate( 'Not included' ), mobileCopy: null },
+				planTwo: { included: false, copy: translate( 'Not included' ), mobileCopy: null },
+			},
+			{
+				featureName: translate( 'Install custom themes' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Install plugins & custom themes' )` *ES Score: 7*